src/components/App/App.jsx

```diff
@@ -1,9 +1,21 @@
+import Project from "../ProjectForm/ProjectDetail";
 import ProjectForm from "../ProjectForm/ProjectForm";
-
-import "./App.scss";
-
+import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
 function App() {
-  return <ProjectForm />;
+  return (
+    <Router>
+      <div className="App">
+        <Switch>
+          <Route path="/project/:id">
+            <Project />
+          </Route>
+          <Route path="/">
+             <ProjectForm />
+          </Route>
```

Comment:
This route-path is wrong and also it's not part of your job.

Reply:
Solved